src/bundle/WysiwygEditorView.scss

```diff
@@ -4,11 +4,13 @@
 
     &__toolbar {
         flex-shrink: 0;
+        padding: var(--g-md-toolbar-padding);
     }
 
     &__editor {
         overflow-y: auto;
         flex-grow: 1;
+        padding: var(--g-md-editor-extra-padding);
```

Comment:
why extra?

Reply:
> why extra?

cause this adds up to the padding of __editor's class nested elements